xbmc/cores/AudioEngine/Utils/AEChannelInfo.cpp

```diff
@@ -121,8 +121,8 @@ void CAEChannelInfo::ResolveChannels(const CAEChannelInfo& rhs)
 void CAEChannelInfo::Reset()
 {
   m_channelCount = 0;
-  for (unsigned int i = 0; i < AE_CH_MAX; ++i)
-    m_channels[i] = AE_CH_NULL;
+  for (AEChannel& channel : m_channels)
```

Comment:
std::fill ?

Reply:
Not sure if  I like, `std::fill(m_channels, m_channels + sizeof(m_channels) / sizeof(AEChannel), AE_CH_NULL);` (m_channels is a C array).
What do you think?

Reply:
less readable, keep the for.